training/two_tower_model_trainer.py

```diff
@@ -0,0 +1,54 @@
+"""
+Copyright 2020 Google LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import numpy as np
+from smart_news_query_embeddings.models.two_tower_model import TwoTowerModel
+from tensorflow.keras.optimizers import Adam
+from bert_model_trainer import BertModelTrainer
+
+class TwoTowerModelTrainer(BertModelTrainer):
+
+    def get_data(self):
+        self.train_ids = np.load('data/all_train_ids.npy')
```

Comment:
use flags for input output path

Reply:
there are 6 of these files. i thought of this class like an experiment run which reads from the pre-defined paths that already have the tokenized sequences